modules/apps/segments/segments-test/src/testIntegration/java/com/liferay/segments/test/SegmentsEntryRetrieverTest.java

```diff
@@ -96,6 +115,16 @@ private SegmentsEntry _addSegmentsEntry(User user) throws Exception {
 			User.class.getName());
 	}
 
+	private boolean _assertContainsEntry(long[] segmentsEntries, long entry) {
```

Comment:
Hey @Stephy6 !!

We don't need a specific method, you could use com.liferay.portal.kernel.util.ArrayUtil.contains :)

Reply:
Oh thats true,  done!